### DISCUSSION TITLE
BoM - How do I seperate References in different lines

### DISCUSSION THREAD

**Discussion by morahor** [INTI-CMNB/KiBot#667] [Q&A]
Im trying to extract the bom into a CVS but it groups those components that share the same value, footprint etc. in the same line.

References,Value,Footprint,Part
C1 C2,6.8pF,C_0603_1608Metric,C
C66 C67,10pF,C_0402_1005Metric,C

for example the C1 and C2 capacitor. How can i separate them into two separate lines but with all the other information attached to both lines?

**Comment by set-soft** [comment]
Do they have any difference in their fields? Why do you need to separate them?

You can use `group_fields` to select the field (or fields) that makes them different. Or you can just use `group_fields: []` so they don't get grouped.

If they are exactly the same then I don't see why you could want to separate them, but you can.

**Reply by morahor** [reply]
Thank you for your swift answer! Your solution worked perfectly. I intended to separate them as the template bom on the website I tried to upload it also separated them. But as you said since there is no difference the unseparated file also worked.
As a sort of follow up question: Is there a way to change the names of the very first line, i.e. "References" to "Designator" "Value" to "Val" etc.?

**Reply by set-soft** [reply]
Use the `columns` option.
Are you creating a BoM for JLCPCB? If this is the case you can use the `JLCPCB` internal template.
